examples/product-detail-page/src/components/ProductView.vue

```diff
@@ -0,0 +1,260 @@
+    currencyCode: "EUR",
+  });
+
+  const productName = computed(() => product.value?.translated?.name);
+  const manufacturer = computed(() => product.value?.manufacturer?.name);
+  const description = computed(() => product.value?.translated?.description);
+  const productNumber = computed(() => product.value?.productNumber);
+  const purchaseUnit = computed(() => product.value?.purchaseUnit);
+  const unitName = computed(() => product.value?.unit?.name);
+</script>
+<template>
+  <div class="md:flex items-start justify-center py-12 2xl:px-20 md:px-6 px-4">
+    <div class="xl:w-2/6 lg:w-2/5 w-80 md:block hidden">
+      <img
+        class="w-full"
+        :alt="`image of ${product?.translated?.name}`"
+        :src="getSmallestThumbnailUrl(product?.cover.media)"
+        :srcset="getSrcSetForMedia(product?.cover.media)"
+      />
+    </div>
+    <div class="md:hidden">
+      <img
+        class="w-full"
+        :alt="`image of ${product?.translated?.name}`"
+        :src="getSmallestThumbnailUrl(product?.cover.media)"
+        :srcset="getSrcSetForMedia(product?.cover.media)"
+      />
+    </div>
+    <div class="xl:w-2/5 md:w-1/2 lg:ml-8 md:ml-6 md:mt-0 mt-6">
+      <div class="border-b border-gray-200 pb-6">
+        <p class="text-sm leading-none text-gray-600 dark:text-gray-300">
+          {{ manufacturer }}
+        </p>
+        <h1
+          class="lg:text-2xl text-xl font-semibold lg:leading-6 leading-7 text-gray-800 dark:text-white mt-2"
+        >
+          {{ productName }}
+        </h1>
+      </div>
+      <div
+        v-if="product?.options?.length > 0"
+        class="py-4 border-b border-gray-200 flex items-center justify-between"
+      >
+        <p class="text-base leading-4 text-gray-800 dark:text-gray-300">
+          {{ product?.options?.[0]?.group.translated?.name }}
+        </p>
+        <div class="flex items-center justify-center">
+          <p class="text-sm leading-none text-gray-600 dark:text-gray-300">
+            {{ product?.options?.[0]?.translated?.name }}
+          </p>
+        </div>
+      </div>
+
+      <div class="mb-12">
+        <div
+          class="inline-flex w-1/3 p-4 pl-0 items-center justify-center font-bold text-2xl drop-shadow"
+        >
+          {{ getFormattedPrice(unitPrice) }}
+          {{ regulationPrice ? "*" : "" }}
+        </div>
+        <button
+          @click="addToCartProxy()"
+          class="mt-4 w-2/3 relative inline-flex items-center justify-center p-0.5 overflow-hidden text-sm font-medium text-gray-900 rounded-lg group bg-gradient-to-br from-cyan-500 to-blue-500 group-hover:from-cyan-500 group-hover:to-blue-500 hover:text-white dark:text-white focus:ring-4 focus:outline-none focus:ring-cyan-200 dark:focus:ring-cyan-800"
+        >
+          <span
+            class="w-full relative px-5 py-2.5 transition-all ease-in duration-75 bg-white dark:bg-gray-900 rounded-md group-hover:bg-opacity-0"
+          >
+            🛍 Add to cart
```

Comment:
I like this icon. Maybe we'd add it to the demo as well? (:

Reply:
added ✅